code/game/gamemodes/meteor/meteor.dm

```diff
@@ -69,6 +69,7 @@
 		universe.OnExit()
 
 	var/datum/universal_state/meteor_storm/meteor_storm = new /datum/universal_state/meteor_storm
+	universe = meteor_storm
```

Comment:
Give `SetUniversalState()` an argument which gets passed to the state's `New()` as an arglist.